icu4j/main/classes/core/src/com/ibm/icu/number/NumberFormatter.java

```diff
@@ -487,9 +487,8 @@
      * <li>AUTO: 0.90, 1.00, 1.10
      * <li>HIDE_IF_WHOLE: 0.90, 1, 1.10
      * </ul>
-     * 
+     *
```

Comment:
unrelated whitespace change

Reply:
Yes, I did a drive-by removal of the last `@provisional` in ICU4J, and Eclipse did its auto-cleanup thing.